CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
 
 ### New Features
 
+- Added support for `session.getOrCreate`
+- Added support for `functions.array_distinct`
+- Added support for alias `Column.getField`
+- Added support for `functions.regexp_extract`
+- Added support for `functions.struct`
+- Added support for `functions.daydiff` which is compatible with spark datediff
+- Added support for `functions.date_add` and `functions.date_sub` to ease common day add/substract operations
+- Added support for `functions.format_number`
+- Added support for `functions.bround`
+
+### Bug Fixes
+
+- Fixed a bug with `strtok_to_array` when a delimiter was passed it was throwing an exception
 - Added support for `explode` function in `snowflake.snowpark.functions`.
 - Added parameter `skip_upload_on_content_match` when creating UDF, UDTF and Stored Procedure using `register_from_file` to skip file uploads to stage in case the files are already present on stage.
```

Comment:
These are also new features. Could you move these back to subsection above

Reply:
Done